src/main/java/core/basesyntax/services/WriteServiceImp.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.services;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class WriteServiceImp implements WriteService {
+    private static final String header = "fruit,quantity";

```

Comment:
refresh knowledge about constant naming convention
```suggestion
    private static final String HEADER = "fruit,quantity";
```